modules/rosa-hcp-aws-privatelink-security-groups.adoc

```diff
@@ -0,0 +1,67 @@
+// Module included in the following assemblies:
+//
+// * rosa_hcp/rosa-hcp-aws-privatelink-creating-cluster.adoc
+[id="rosa-hcp-aws-privatelink-security-groups_{context}"]
+= Configuring AWS security groups to access the API
+
+With {hcp-title} private clusters, the AWS PrivateLink endpoint exposed in the customer's VPC has a default security group and is applied limiting access to the PrivateLink endpoint to only those resources that exist within the VPC, or that present with an IP address associated with the VPC CIDR range. In order to grant access to any entities outside of the VPC, either through VPC peering, transit gateway, etc., you will need to create and attach another security group to the PrivateLink endpoint granting the necessary access.
```

Comment:
I think an extra noun would be clearer here, up to you:
```suggestion
With {hcp-title} private clusters, the AWS PrivateLink endpoint exposed in the customer's VPC has a default security group and is applied limiting access to the PrivateLink endpoint to only those resources that exist within the VPC, or resources that present with an IP address associated with the VPC CIDR range. In order to grant access to any entities outside of the VPC, either through VPC peering, transit gateway, etc., you will need to create and attach another security group to the PrivateLink endpoint granting the necessary access.
```
Not sure if "resources" is the right noun, though